internal/commands/pull/command_test.go

```diff
@@ -478,18 +478,6 @@ Successfully pulled app down: app
 			assert.Nil(t, err)
 			defer console.Close()
 
-			doneCh := make(chan struct{})
```

Comment:
if this test is no longer interactive, then let's move the `ui` back to `mock.NewUI` (rather than the `V10X` variant)